bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -366,9 +368,14 @@ Dimmer              iLR_dimmer                  "Dimmer [%.0f %%]"            (g
 
 Rollershutter       iLR_shutter                 "Shutter [%.0f %%]"           (gShutters, gLivingRoom) { channel="openwebnet:bus_automation:mybridge:LR_shutter:shutter", ga="Blinds", homekit = "Window" }
 
-Number:Power        iCENTRAL_Ta                 "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:power" }
-Number:Power        iCENTRAL_Tb                 "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:power" }
-
+Number:Power        iCENTRAL_Ta_pow              "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:power" }
+Number:Power        iCENTRAL_Tb_pow              "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:power" }
+Number              iCENTRAL_Ta_day              "Energy Day  [%.0f KW]"                         { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:totalizerDay" }
+Number              iCENTRAL_Tb_day              "Energy Day [%.0f KW]"                          { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:totalizerDay" }
+Number              iCENTRAL_Ta_month            "Energy Month [%.0f KW]"                        { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:totalizerMonth" }
+Number              iCENTRAL_Tb_month            "Energy Month [%.0f KW]"                        { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:totalizerMonth" }
```

Comment:
- formatting should be %.1f if this how it's defined for the channel
- item type should be Number:Energy, and unit of measure should probably set automatically (so removed from formatting here). In any case it's not KW but kWh.
- channel names must be changed accordingly to new channel names

Reply:
ok

Reply:
ok